terraform/container-app/contentful.tf

```diff
@@ -0,0 +1,22 @@
+//Double check this is the right thing
+data "azurerm_cdn_frontdoor_endpoint" "app" {
+  name                = "s190d01-plantech-container-app-url"
+  profile_name        = "s190d01-plantech-cdnwaf"
+  resource_group_name = "s190d01-plantech"
+}
+
+resource "null_resource" "health_check" {
+  triggers = {
+    api_key_change         = random_password.api_key_value.result
+    url_change             = data.azurerm_cdn_frontdoor_endpoint.app.host_name
+    management_token       = var.contentful_management_token
+    contentful_environment = azurerm_key_vault_secret.vault_secret_contentful_environment.value
+    contentful_space       = azurerm_key_vault_secret.vault_secret_contentful_spaceid.value
+    webhook_url            = var.contentful_webhook_endpoint
+    should_upsert          = var.contentful_upsert_webhook == true ? timestamp() : var.contentful_upsert_webhook
```

Comment:
Just making sure I follow - so these are all the values that, if they change, should recreate the null resource, which itself may or may not recreate the webhook, and one of those triggers is should_upsert
So, if `contentful_upsert_webhook` is true, it uses the timestamp, which will always be different, so is guaranteed to trigger it, and otherwise, just use the bool, so if it goes from true to false, that would also trigger it?

Reply:
That's the badger mate